docs/index.md

```diff
@@ -16,6 +16,7 @@ redirect_from:
 
 ### Apache Fluo Recipes documentation
 
+* [1.2][recipes-1.2] - March 1, 2018
```

Comment:
Can someone validate this date? I got it from the 1.2 tag in the fluo-recipes repo.

Reply:
The official date for the release is technically the date of distribution, which is effectively the date it was uploaded to the SVN-dist mirror system. That date is `March 6, 2018`.